### DISCUSSION TITLE
[blocks]: Please bring back the blocks you replaced with sidebar blocks

### DISCUSSION THREAD

**Discussion by robert-cardillo** [shadcn-ui/ui#5962] [Blocks Request]
### Description

The dashboard blocks from before you introduced sidebar component were very useful. Please bring them back.

https://web.archive.org/web/20240702114432/https://ui.shadcn.com/blocks

**Comment by govinda898931** [comment]
Hi,
I believe the components are still there. You can check through the waybackmachine for their names and install them using
npx shadcn@latest add sidebar-05

replace sidebar-05 with any of the blocks that you see. They are just archived from the webpage. Still available to install

**Comment by robert-cardillo** [comment]
Never mind. Here you go:

[https://shadcn.usame.link/blocks](https://shadcn.usame.link/blocks)
